docs/source/index.md

```diff
@@ -153,11 +153,9 @@ See [Google Scholar](https://scholar.google.de/scholar?oi=bibs&hl=en&authuser=1&
 :maxdepth: 1
 :hidden:
 
-installation
 learning
-Examples <https://docs.pymc.io/projects/examples/en/latest/>
-api
+API <api>
 community
 contributing/index
-about/index
+About <about/index>
```

Comment:
I have modified this to "About" only, but with "About PyMC" as the title in the page. It is still unclear what will happen with https://github.com/pymc-devs/pymc.io which might end up hosting the about page with the testimonials, blog posts and history. The theme only allows that external links have the []/ icon if they are at the end of the navbar. Keeping it here for now would mean minimal change if eventually we "outsource" the about page to pymc.io.

Reply:
Related to https://github.com/pymc-devs/pymc.io/pull/4

Reply:
This is now an external link